fdm-app/app/components/blocks/soil/bulk-upload-form.tsx

```
@@ -0,0 +1,189 @@
+        const formData = new FormData()
+        for (const file of files) {
+            formData.append("soilAnalysisFile", file)
+        }
+
+        fetcher.submit(formData, {
+            method: "POST",
+            encType: "multipart/form-data",
+        })
+    }
+
+    useEffect(() => {
```

Biome (2.3.13)

[error] This hook does not specify its dependency on isUploading.

This dependency is being used here, but is not specified in the hook dependency list.

React relies on hook dependencies to determine when to re-compute Effects.
Failing to specify dependencies can result in Effects not updating correctly when state changes.
These "stale closures" are a common source of surprising bugs.
Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)